lib/src/providers/VernaProvider.spec.tsx

```diff
@@ -0,0 +1,92 @@
+import {
```

Comment:
Instead to have `VernaProvider.spec.tsx` and `VernaProvider.tsx` at the root of `providers` directory please create a folder `VernaProvider` then move those files within like we did for `components/VernaForm` for example.